languages.yaml

```diff
@@ -138,6 +138,38 @@ Assembly:
       - yellow
       - green
     chip: "#6E4C13"
+Astro:
+  type: programming
+  ascii: |                                               
+    {0}+++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    {0}++++++++++++++++++++++*{1}@@@@@@@@@{0}#++++++++++++++++++++++
+    {0}++++++++++++++++++++++{1}@@@@@@@@@@@{0}#+++++++++++++++++++++
+    {0}+++++++++++++++++++++%{1}@@@@@@@@@@@@{0}+++++++++++++++++++++
+    {0}++++++++++++++++++++#{1}@@@@@@#@@@@@@{0}%++++++++++++++++++++
+    {0}++++++++++++++++++++{1}@@@@@@#+*{1}@@@@@@{0}*+++++++++++++++++++
+    {0}+++++++++++++++++++%{1}@@@@@@{0}++++{1}%@@@@@@{0}+++++++++++++++++++
+    {0}++++++++++++++++++*{1}@@@@@@{0}*+++{1}+@@@@@@{0}#++++++++++++++++++
+    {0}++++++++++++++++++{1}@@@@@@{0}%++++++{1}#@@@@@@{0}++++++++++++++++++
+    {0}+++++++++++++++++#{1}@@@@@@{0}*++++++{1}@@@@@@{0}%+++++++++++++++++
+    {0}+++++++++++++++++{1}@@@@@@{0}#+++++++*{1}@@@@@@{0}*++++++++++++++++
+    {0}++++++++++++++++%{1}@@@%%{0}#+++++++++{1}##%@@@@{0}++++++++++++++++
+    {0}++++++++++++++++{1}#*{0}+++++++++++++++++++{1}*#*{0}+++++++++++++++
```

Comment:
Rather than using black and white it would be better to just use white and empty space (or no white background) for better compatibility with light terminals.

Reply:
That's a really good idea, I'll change it.